src/containers/lists/BidList/BidList.tsx

```diff
@@ -131,7 +131,7 @@ const BidList: React.FC<BidListProps> = ({
 										{Number(
 											ethers.utils.formatEther(bid.amount),
 										).toLocaleString()}{' '}
-										{paymentTokenInfo.name}
+										{paymentTokenInfo.symbol}
```

Comment:
<img width="899" alt="Screenshot 2022-07-14 at 23 04 21" src="https://user-images.githubusercontent.com/39112648/179097787-53b8baa0-0388-484a-a201-b61c2ca3c0a1.png">

Seems to be an issue in the user profile

Reply:
What's the issue here?